src/components/MerchantPage/sample-merchant.ts

```diff
@@ -1,3 +1,5 @@
+import avatar from './images/female.svg'
```

Comment:
@ArtyEmsee this is what you'd put into your database in order to make the default avatar show up